cmd/abapEnvironmentRunATCCheck.go

```diff
@@ -63,7 +63,7 @@ func abapEnvironmentRunATCCheck(options abapEnvironmentRunATCCheckOptions, telem
 		resp, err = triggerATCRun(options, details, &client)
 	}
 	if err == nil {
-		err = fetchAndPersistATCResults(resp, details, &client, &fileUtils, options.AtcResultsFileName, options.GenerateHTML)
+		err = fetchAndPersistATCResults(resp, details, &client, &fileUtils, options.AtcResultsFileName, options.GenerateHTML, options.FailOnSeverity)
 	}
 	if err != nil {
 		log.Entry().WithError(err).Fatal("step execution failed")
```

Comment:
```suggestion
		if err = fetchAndPersistATCResults(resp, details, &client, &fileUtils, options.AtcResultsFileName, options.GenerateHTML, options.FailOnSeverity); err != nil {
			log.Entry().WithError(err).Fatal("step execution failed")
		}
```
if the returning value is just an error it can also be written like this so imho readability is a bit better.